tests/components/post_view/commented_on.test.jsx

```diff
@@ -58,6 +162,17 @@ describe('components/post_view/CommentedOn', () => {
 
         expect(baseProps.actions.updateSearchTerms).toHaveBeenCalledTimes(1);
         expect(baseProps.actions.updateSearchTerms).toHaveBeenCalledWith(baseProps.displayName);
+        expect(baseProps.actions.showSearchResults).toHaveBeenCalledTimes(1);
+    });
+
+    test('Should trigger search with ', () => {
```

Comment:
nit: "with" what?

Reply:
Oops my bad it supposed to say ovveride_username